packages/cli-repl/bin/mongosh.js

```diff
@@ -1,4 +1,5 @@
-#!/usr/bin/env node
+#!/bin/sh
```

Comment:
Nice trick! Is this how it was done before?

Reply:
yes it was!